test/detectors/SuspiciousMessageMode.tact

```diff
@@ -0,0 +1,112 @@
+// Define custom constants
+const CustomSendRemainingValue: Int = 64;
+const CustomSendAllBalance: Int = 128;
+const CustomSendIgnoreErrors: Int = 1;
+
+contract SendParametersTestContract {
+    // Correct usage: should not trigger any warnings
+    fun correctUsage() {
+        send(SendParameters{
+            to: sender(),
+            value: 0,
+            mode: CustomSendRemainingValue
+        });
+    }
+
+    // Duplicate flag usage: should trigger a warning about flags used multiple times
+    fun duplicateFlagUsage() {
+        send(SendParameters{
+            to: sender(),
+            value: 0,
+            mode: CustomSendRemainingValue | CustomSendRemainingValue
+        });
+    }
+
+    // Invalid operator usage: should trigger a warning about using '+' instead of '|'
+    fun invalidOperatorUsage() {
+        send(SendParameters{
+            to: sender(),
+            value: 0,
+            mode: CustomSendRemainingValue + CustomSendAllBalance
+        });
+    }
+
+    // Integer literal usage: should trigger a warning about using integer literals
+    fun integerLiteralUsage() {
+        send(SendParameters{
+            to: sender(),
+            value: 0,
+            mode: 64 // Integer literal instead of symbolic constant
+        });
+    }
+
+    // Correct combination usage: should not trigger any warnings
+    fun correctCombinationUsage() {
+        send(SendParameters{
+            to: sender(),
+            value: 0,
+            mode: CustomSendRemainingValue | CustomSendAllBalance | CustomSendIgnoreErrors
```

Comment:
I would add tests with the existing values from the docs: https://docs.tact-lang.org/book/message-mode/